MessageBusReader/Program.cs

```diff
@@ -15,6 +15,8 @@ class Program
         private static ServiceBusClient _client;
         private static ServiceBusProcessor _processor;
 
+        private const string QueueName = "error";
+
```

Comment:
Now that we have several different error queues I though we might benefit from a global variable to switch which queue we are processing.